DigitalLearningSolutions.Data.Migrations/202201171115_UndoPreviousConstraintMigrationsForCompetencyAssessmentQuestionRoleRequirements.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202201171115)]
+    public class UndoPreviousConstraintMigrationsForCompetencyAssessmentQuestionRoleRequirements : Migration
+    {
+        public override void Up()
+        {
+            if (Schema.Table("CompetencyAssessmentQuestionRoleRequirements").Constraint(
+                "IX_CompetencyAssessmentQuestionRoleRequirements_SelfAssessmentID_CompetencyID"
+            ).Exists())
+            {
+                // This undoes the changes from 202201120821_ChangeUniqueConstraintsOnCompetencyAssessmentQuestionRoleRequirements

```

Comment:
I'll be picky and suggest instead "This undoes the **unwanted** changes..."

Reply:
That's an appropriate suggestion, I've added it.